docs/bestPractises/pullRequest-start11.md

```diff
@@ -0,0 +1,16 @@
+# Best practises: Pull requests
+This document describes best practises when dealing with pull requests, that were learned within the start11 team.
+
+## Creation of the pull request
+TBD
+
+### Basic configuration
+TBD
+
+### Configuration of bots / checks
+TBD
+
+### Reviewing and Merging
+TBD
+
+**[back to pull requests](/docs/development.md#pull-requests)**
```

Comment:
This is my take on the pull request overview for Team Appengers. I've done some screenshots, added them here, I hope that's okay for you. 🙂 Feel free to add any comments on this.

```## Creation of the pull request
### Name
Set the git commit message as the name of the pull request. When merging this is the suggested name for the commit message, which lets you easily confirm the PR instead of setting a proper git commit message.

#### Examples
- fix: reflash feature issues
- feat: add translation for contact and legal notice page

### Description
The description is already outlined by our template, which can be found in any of our repositories.

Basically it contains all information about:
- Short description of the PR
- Jira ticket and subtask ticket
- What is needed to reproduce the bug respectively the fix or what steps are needed to test the feature
- A list of devices and simulators the build was tested on
- A checklist at the end to make sure all necessary steps are done (e.g. changelog entry, unit tests pass, etc.)

#### Examples
![](../../files/screenshots/pullrequest_bestpractise_appengers.png)

### Configuration of bots / checks
Checks that have have to pass are Bitrise (the PR check workflow) and Danger on iOS, SonarCloud Bot on Android.

![](../../files/screenshots/pullrequest_bestpractise_appengers_android_bot.png)
![](../../files/screenshots/pullrequest_bestpractise_appengers_ios_bot.png)

### Reviewing and Merging
We have applied the following rules for this project:

- Checkout the branch holding the commits
- Review the code, run and test it locally, run the unit tests locally
- Do not squash commits when merging
```

[best_practices_screenshots.zip](https://github.com/nanogiants/nanogiants-docs/files/12064947/best_practices_screenshots.zip)